### PR TITLE
Preparing documentation for 1.1 release (#457)

### PR DESCRIPTION
**Pull request by maxim-tschumak**
Related to #457.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,9 +65,8 @@ cd zally
 
 # Disable authentication and start a local version of Zally server
 cd server
-echo "spring.profiles.active=dev" >> application.properties
 ./gradlew clean build
-./gradlew bootRun > /dev/null &
+./gradlew bootRun
```
Comment (netme) [inline README.md]: That will block terminal and next command won't be available

File: README.md [hunk 2]
```diff
@@ -24,38 +24,38 @@ Minimalistic API linter of awesomeness, bringing order to a sea of APIs.
...
-- improve CLI user experience
-- improve web UI
-- use better integration testing approaches
+For [version 1.2](https://github.com/zalando-incubator/zally/milestone/3) we focus on these main areas:
 
-Feel free to contribute on related issues for version 1.1.
+- Consistency with [Zalando RESTful API Guidelines](http://zalando.github.io/restful-api-guidelines/)
+- Referential integrity with [Zalando RESTful API Guidelines](http://zalando.github.io/restful-api-guidelines/)
+- New rules
+- Improve quick start
```
Comment (netme) [inline README.md]: What exactly does it mean? :-)

File: README.md (hunk 1 above)
Comment (maxim-tschumak) [inline README.md]: done

File: README.md (hunk 2 above)
Comment (maxim-tschumak) [inline README.md]: -> `New quick start script and better integration testing approaches`

**Comment by maxim-tschumak** [comment]
@netme please have another look
:+1:

**Comment by netme** [comment]
👍

**Comment by maxim-tschumak** [comment]
:+1:

**Pull request merged**